t/plugin/log-rotate2.t

```diff
@@ -25,23 +25,6 @@ no_root_location();
 add_block_preprocessor(sub {
     my ($block) = @_;
 
-    my $user_yaml_config = <<_EOC_;
-apisix:
-  node_listen: 1984
-  admin_key: null
-
-plugins:                          # plugin list
-  - log-rotate
-
-plugin_attr:
-  log-rotate:
-    interval: 1
-    max_kept: 3
-    enable_compression: true
-_EOC_
-
-    $block->set_value("yaml_config", $user_yaml_config);
```

Comment:
We can check if yaml_config exists before setting it

Reply:
updated